inc/Engine/Optimization/RUCSS/Controller/UsedCSS.php

```diff
@@ -215,7 +215,7 @@ public function treeshake( string $html ): string {
 
 		if ( empty( $used_css ) ) {
 			$add_to_queue_response = $this->add_url_to_the_queue( $url, $is_mobile );
-			if ( false === $add_to_queue_response ) {
+			if ( false === $add_to_queue_response || ! isset( $add_to_queue_response['contents'] ) || ! isset( $add_to_queue_response['contents']['jobId'] ) || ! isset( $add_to_queue_response['contents']['queueName'] ) ) {
```

Comment:
You can pass multiple variables to the same `isset()` call, that will make it more readable